#traffic signal info
float32[] traffic_top_left_x
float32[] traffic_top_left_y
float32[] traffic_bottom_right_x
float32[] traffic_bottom_right_y
float32[] traffic_pval

string[] traffic_color_label
float32[] traffic_color_pval

#pedestrian signal info
float32[] pedestrian_top_left_x
float32[] pedestrian_top_left_y
float32[] pedestrian_bottom_right_x
float32[] pedestrian_bottom_right_y
float32[] pedestrian_pval

string[] pedestrian_color_label
float32[] pedestrian_color_pval